documentation/modules/ref-standard-infra-config-override-probe-timeout.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// assembly-infrastructure-configuration.adoc
+
+[id='ref-standard-infra-config-override-probe-timeout-{context}']
+= Overriding the readiness probe timing for standard infrastructure configuration
+
+You can override the default value for the readiness probe timing.
```

Comment:
How about.. "The following example shows how to override some default values of the readiness probe timeout for the broker resource.". 

I also wonder if it is worth adding to this section an illustration of overriding the liveness probe for router resources.